src/MediaWiki/Search/ProfileForm/Forms/CustomForm.php

```diff
@@ -66,7 +66,7 @@ public function __construct( WebRequest $request ) {
 	/**
 	 * @since 3.0
 	 *
-	 * @return
+	 * @return array
 	 */
 	public function getParameters() {
```

Comment:
We can put a type declaration here and drop the `@return` doc

Reply:
do you mean like `: array {`?

Reply:
Thanks, done.